tests/e2e/run-scanner-v4.bats

```diff
@@ -68,17 +68,167 @@ setup() {
     fi
 
     test_case_no=$(( test_case_no + 1))
+
+    export MAIN_IMAGE_TAG=${MAIN_IMAGE_TAG:-}
+    info "Overriding MAIN_IMAGE_TAG=$MAIN_IMAGE_TAG"
+
+    export ROX_SCANNER_V4=true
 }
 
 teardown() {
-    local namespaces=( "stackrox" "$CUSTOM_CENTRAL_NAMESPACE" "$CUSTOM_SENSOR_NAMESPACE" )
-    for namespace in "${namespaces[@]}"; do
-        if kubectl get ns "${namespace}" >/dev/null 2>&1; then
-            run remove_existing_stackrox_resources "${namespace}"
+    local central_namespace=""
+    local sensor_namespace=""
+    local namespaces=( )
+
+    if "${ORCH_CMD}" get ns "stackrox" >/dev/null 2>&1; then
+        central_namespace="stackrox"
+        sensor_namespace="stackrox"
+        namespaces=( "stackrox" "${namespaces[@]}" )
+    fi
+    if "${ORCH_CMD}" get ns "${CUSTOM_CENTRAL_NAMESPACE}" >/dev/null 2>&1; then
+        central_namespace="${CUSTOM_CENTRAL_NAMESPACE}"
+        namespaces=( "${central_namespace}" "${namespaces[@]}" )
+    fi
+    if "${ORCH_CMD}" get ns "${CUSTOM_SENSOR_NAMESPACE}" >/dev/null 2>&1; then
+        sensor_namespace="${CUSTOM_SENSOR_NAMESPACE}"
+        if [[ "${central_namespace}" != "${sensor_namespace}" ]]; then
+            namespaces=( "${sensor_namespace}" "${namespaces[@]}" )
         fi
+    fi
+
+    if [[ -z "${BATS_TEST_COMPLETED:-}" ]]; then
+        # Test failed, collect some analysis data.
+        info "Pods in namespace ${central_namespace}:"
+        "${ORCH_CMD}" -n "${central_namespace}" get pods
```

Comment:
Could you please add `describe` for each pod? Rr at least for centra;/scanner pods. This might be really helpful for debugging

Reply:
done, commit soon to be pushed